simulator/src/testbench.js

```
@@ -820,7 +812,11 @@ function bindIO(data, scope) {
     });
 
     data.groups[0].outputs.forEach((dataOutput) => {
-        outputs[dataOutput.label.trim()] = scope.Output.find((simulatorOutput) => simulatorOutput.label.trim() === dataOutput.label.trim());
+        let matchOutput = scope.Output.find((simulatorOutput) => simulatorOutput.label.trim() === dataOutput.label.trim());
+        if (matchOutput === undefined) {
+    matchOutput = scope.DigitalLed.find((simulatorOutput) => simulatorOutput.label.trim() === dataOutput.label.trim());
```

ESLint

[error] Expected indentation of 12 spaces but found 4.

(indent)